rfcs/0000-User&HostIntegrationRFC.md

```diff
@@ -0,0 +1,254 @@
+
+
+
+### Proposed New Fields for Host object
+Field | Type | Example | Description
+--- | --- | --- | ---
+host.os.build	| keyword		| 22F66   | Host OS Build information
+host.installed_extensions 	| keyword	  | Nested objects	  | List of installed extensions along with their metadata
+host.installed_applications	| keyword	    | Nested objects	  | List of installed applications along with their metadata
+
+
+
+<!--
+Stage 2: Add or update all remaining field definitions. The list should now be exhaustive. The goal here is to validate the technical details of all remaining fields and to provide a basis for releasing these field definitions as beta in the schema. Use GitHub code blocks with yml syntax formatting, and add them to the corresponding RFC folder.
+-->
+
+## Usage
+
+<!--
+Stage 1: Describe at a high-level how these field changes will be used in practice. Real world examples are encouraged. The goal here is to understand how people would leverage these fields to gain insights or solve problems. ~1-3 paragraphs.
+-->
+
+* As part of Entity Analytics, we are ingesting metadata about [Users](https://github.com/elastic/security-team/issues/4163) and [Hosts](https://github.com/elastic/security-team/issues/4044) from various external vendor applications. We are storing all ingested metadat in ElasticSearch inde. After we map these fields to ECS, we will enrich these ingested events for risk scoring scenarios (e.g. of context enrichments) and detecting advanced analytics (UBA) use cases. 
+
+* This schema will be used to persist `Observed` (queried) entities from ingested security log dataset in an Entity store. This entity store can be further extended to meet broaded overall Asset Mangagement needs.
+
+* We also envision additional enrichment use cases for existing prebuilt detection rules down the line.
+
+
+## Source data
+
+<!--
+Stage 1: Provide a high-level description of example sources of data. This does not yet need to be a concrete example of a source document, but instead can simply describe a potential source (e.g. nginx access log). This will ultimately be fleshed out to include literal source examples in a future stage. The goal here is to identify practical sources for these fields in the real world. ~1-3 sentences or unordered list.
+-->
+
+There are many sources of asset inventory repositories. In the mid-term we are planning to ingest data from the following applicaiton providers:
+
+### User (Identity) repository sources:
+*  Azure Active Directory
+* Active Directory DS
+* Okta 
+* Workday
+* GSuite 
+* GitHub
+
+### Host repository sources:
+* Azure Active Directory
+* Jamf 
+* Active Directory DS
+* MS Intune
+* ServiceNow Asset CMDB
+
+Mapping of application fields from individual providers are documented here: [User](https://docs.google.com/spreadsheets/d/1ZHGQPGVr35vLZB3DcPnXWxLTkHad_S0YrhtTKO_LL4o/edit#gid=1940810377), [Host-TBD]().
```

Comment:
Like any linked GitHub issues, any other linked resources should be publicly viewable for the community.